modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/sqllogic/Query.java

```diff
@@ -327,13 +327,19 @@ private void checkResultTuples(ScriptContext ctx, List<List<?>> res) {
                 String expectStrValRaw = expectedRow.get(j);
                 String expectStrVal = ctx.replaceVars(expectStrValRaw);
 
-                checkEquals(ctx,
-                        "Not expected result at: " + posDesc
-                                + ". [row=" + i + ", col=" + j
-                                + ", expected=" + expectStrVal + ", actual=" + resultToString(ctx, row.get(j)) + ']',
-                        expectStrVal,
-                        row.get(j)
-                );
+                try {
+                    checkEquals(ctx,
+                            "Not expected result at: " + posDesc
+                                    + ". [row=" + i + ", col=" + j
+                                    + ", expected=" + expectStrVal + ", actual=" + resultToString(ctx, row.get(j)) + ']',
+                            expectStrVal,
+                            row.get(j)
+                    );
+                } catch (AssertionError ex) {
+                    AssertionError extended = new AssertionError("Returned results: " + res);
+                    extended.addSuppressed(ex);
```

Comment:
Why do you suppress original exception instead of using cause?

Reply:
Maybe, it is better log a error instead?

Reply:
got it, fixed

Reply:
i think about logging too, but as for me - assertion error is better to parse than log, but if you insist - i can change it